README.md

```diff
@@ -69,6 +69,43 @@ This assumes you make your build dir a subdirectory of the ceph.git
 checkout.  If you put it elsewhere, just replace .. above with a
 correct path to the checkout.
 
+To build only certain targets use:
+
+    $ make [target name]
+
+To install:
+
+    $ make install
+ 
+CMake Options
+-------------
+
+If you run the `cmake` command by hand, there are many options you can
```

Comment:
@ivancich that's a good question, the -D options are not just limited to the options we define in CMakeLists.txt with the option() function, but also to variables defined by CMake (ex: CMAKE_BUILD_TYPE, CMAKE_C_FLAGS, etc.).  So I don't think it would be good to have an exhaustive list, but what do you think about a list of 10-20 very important ones?